packages/amplify-graphql-transformer-core/package.json

```diff
@@ -38,6 +38,7 @@
         "@aws-cdk/aws-dynamodb": "~1.124.0",
         "@aws-cdk/aws-ec2": "~1.124.0",
         "@aws-cdk/aws-efs": "~1.124.0",
+        "@aws-cdk/aws-elasticsearch": "^1.124.0",
```

Comment:
@akshbhu this has the wrong version prefix. It must be `~`. See the changes in yarn.lock for why

Reply:
done